src/radtest/memtest.c

```diff
@@ -0,0 +1,255 @@
+/*
+ * mramtests.c
+ *
+ *  Created on: 16.08.2022
+ *      Author: Robert
+ */
+
+#ifndef RADTEST_MRAMTESTS_C_
+#define RADTEST_MRAMTESTS_C_
+
+#define RADTST_SEQ_READCHECKS_SECONDS				 15			// Initiate all read checks every n seconds
+#define RADTST_SEQ_WRITECHECKS_SECONDS				 60			// Initiate all write checks every n seconds
+#define RADTST_SEQ_COUNTERPRINT_SECONDS				120			// Initiate printout of current counters/errors
```

Comment:
@WolfgangTreb 
@andreassinn 
@ Abbas 

Please Verify and confirm with which frequency memory tests should be run ! 

What I heard from Abbas today is that we need to make debug prints more frequent  (4-6s) ? But I am not familiar with RadTest details !

Reply:
We agreend to speed up a little because the test will just take about 3 hours (our last test at SL was for several days) so a reading of 4 seconds od the Supervision watchdog and i2c sensros is planed